interface/src/app/gene/searchFew/searchFew.js

```diff
@@ -14,7 +14,8 @@ angular.module('adage.gene.searchFew', [
     }],
     restrict: 'E',
     scope: {
-      organism: '=',
+      organism: '@',
+      mLModel: '@',
```

Comment:
Similar to my comment elsewhere, we could make this an `mlModel` and I think you'd be able to refer to `ml-model`s in the `.tpl.html` files.